introduction.md

```diff
@@ -4,3 +4,7 @@ Introduction
 My name is Manuel, I'm a COBOL programer from several years ago. I'm just started to learnt GITHUB.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?